README.md

```diff
@@ -53,26 +53,57 @@ For details on all functionality of this library, see the [GoDoc][godoc] documen
 
```

Comment:
The heading structure of the document is a bit off:

```
## Getting Started
<no content>
### Installation
<content>
## Examples
<content>
### Read a secret
<content>
### Write a secret
<content>
### Generate a secret
<content>
```

I'd suggest making it something like:

```
## Usage (or getting started)
<`go get` instructions>
<import (AND client construct instructions?)>
<notice about getting an account (which should include that `secrethub.NewClient` auto-loads the credential)>

### Examples

#### Read a secret
<code>
#### Write a secret
<code>
#### Generate a random secret
<code>
```

Reply:
https://github.com/secrethub/secrethub-go/pull/141